relay-server/src/services/store.rs

```diff
@@ -244,6 +246,20 @@ impl StoreService {
                         item,
                     )?;
                 }
+                ItemType::UserReportV2 => {
+                    if use_feedback_ingest_v2 {
```

Comment:
With this, you can make sure that there's an `error` log if there's an unexpected feedback item in the envelope.

```suggestion
                ItemType::UserReportV2 if use_feedback_ingest_v2 => {
```